components/base/kinematicbase/kinematics_test.go

```diff
@@ -44,12 +49,14 @@ func TestWrapWithDifferentialDriveKinematics(t *testing.T) {
 	testCases := []struct {
 		geoType spatialmath.GeometryType
 		success bool
+		linVel  float64
+		angVel  float64
 	}{
-		{spatialmath.SphereType, true},
-		{spatialmath.BoxType, true},
-		{spatialmath.CapsuleType, true},
-		{spatialmath.UnknownType, true},
-		{spatialmath.GeometryType("bad"), false},
+		{spatialmath.SphereType, true, 10.1, 20.2},
+		{spatialmath.BoxType, true, 1, 2},
+		{spatialmath.CapsuleType, true, 0, -10.5},
+		{spatialmath.UnknownType, true, -10.9, 0},
+		{spatialmath.GeometryType("bad"), false, 99.9, 88.8},
```

Comment:
I had intended this test to just test geometries, instead of doing this lets add a different subtest in this function to make sure the veolcity gets passed through correctly

Reply:
Done!